src/System.Management.Automation/engine/NullLiteral.cs

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace System.Management.Automation.Internal
+{
+    /// <summary>
+    /// This singleton provides a way to identify when a null literal was
+    /// passed into a cmdlet that has explicit support for null literals
+    /// (e.g. Where-Object when used with -is/-isnot).
+    /// </summary>
+    internal class NullLiteral
+    {
+        #region public_property
```

Comment:
Cut and paste? These regions seem unnecessary.

Reply:
Yes, this was copied/pasted from the `NullString` class. Regions removed in next commit.